devnet/substrate_bob.Dockerfile

```diff
@@ -1,14 +1,11 @@
-# This is the build stage for Substrate. Here we create the binary.
-FROM parity/polkadot:v0.9.10 as polkadot
+FROM parity/polkadot:v0.9.10 AS polkadot
 FROM golang:1.17
 
 ARG key
 ARG CHAIN=cross-client
 ARG DD_API_KEY=somekey
 ARG METRICS_NAMESPACE=substrate.local.devnet
 
-ENV key=${key}
-ENV CHAIN=${CHAIN}
 ENV DD_API_KEY=${DD_API_KEY}
```

Comment:
I might be repeating myself, but shouldn't the API key be injected at runtime and not build? Sounds risky to have a key there

Reply:
This image is stored inside a private container registry, but @timwu20 can provide more context about the need of this key inside the container environment

Reply:
ohh yeah that's true so never mind. Although it should be possible to have at build time, and then injected at runtime (and not stored in the image). Anyway for another time!